helm-chart/renku-notebooks/values.yaml

```diff
@@ -50,6 +57,23 @@ image:
 # turns on flask debug mode
 debug: true
 
+# oidc settings
+oidc:
+  clientId: renku-jupyterserver
+  clientSecret:
+  token_url:
+  auth_url:
+  allowUnverifiedEmail: false
+
+session_ingress:
```

Comment:
```suggestion
sessionIngress:
```

Reply:
Noticed this when creating https://github.com/SwissDataScienceCenter/sdsc-dev-docs/pull/96 that the notation here deviates a bit from the usual chart ingress notation that you get for example from `helm create`: 
 
 ```
 ingress:
  enabled: false
  annotations: {}
    # kubernetes.io/ingress.class: nginx
    # kubernetes.io/tls-acme: "true"
  hosts:
    - host: chart-example.local
      paths: []
  tls: []
  #  - secretName: chart-example-tls
  #    hosts:
  #      - chart-example.local
  ```

Reply:
🤦  sorry yeah I will fix that. It seems I am somehow incapable of properly switching between snake, kebab and camel case.

Reply:
@ableuler I found more cases in the `values.yaml` where I dont use camel case but I should. I will fix all of these with one PR.

Reply:
Using camelCase in yaml is definitely good, but my point was at leas as much about making hosts a list instead of single entry and expect the annotations as object rather than a string-serialized version of it.

Reply:
Actually, for the host vs hosts, I'd like to correct myself and keep it singular.